eliot-service/eliot/app.py

```diff
@@ -39,6 +42,15 @@
 STATICROOT_DIR = str(Path(__file__).parent / "static")
 
 
+SCRUB_RULES_ELIOT = [
+    Rule(
+        path="request.headers",
+        keys=["X-Forwarded-For", "X-Real-Ip"],
+        scrub="scrub",
+    ),
+]
```

Comment:
Eliot has no user accounts, so the only sensitive data it's got is infrastructure-related.